privacyidea/static/components/token/views/token.list.html

```diff
@@ -130,7 +133,22 @@
             user_details_in_tokenlist">
                 <a ui-sref="config.resolvers.list">{{ token.resolver }}</a>
             </td>
+            <td ng-show="!checkPolicyEmpty(token_rollover)">
```

Comment:
Same here, even if the button is disabled it might confuse people.

Reply:
done in 90b60853d1020aa950c45f486257760931a1b290